Makefile

```diff
@@ -31,31 +33,17 @@ GORELSNAP = #--snapshot # uncomment --snapshot if you want to do a dry run.
 SKIP = #\# # To skip push to docker and github remove # in front of #
 GORELDEBUG = #--debug # uncomment --debug for verbose goreleaser output
 
-ifeq ($(OS),Windows_NT)
-	GOPATH = $(shell cygpath -u $(shell go env GOPATH))
-else
-	GOPATH = $(shell go env GOPATH)
-endif
-
-GOBIN = $(GOPATH)/bin
-
-ifdef TRAVIS_BUILD_NUMBER
-	# Upper bound for travis test timeout
-	TIMEOUT = 3600s
-else
-ifndef TIMEOUT
-	# You should be able to specify your own timeout, but by default we'll use the same bound as travis
-	TIMEOUT = 3600s
-endif
-endif
+# Upper bound for travis test timeout
```

Comment:
We don't use travis in master anymore right? Should the comment be changed to CircleCI?

Reply:
Done.